pyproject.toml

```diff
@@ -151,6 +151,9 @@ manylinux-x86_64-image = "manylinux2014"
 manylinux-aarch64-image = "manylinux2014"
 musllinux-x86_64-image = "musllinux_1_1"
 
+[tool.cibuildwheel.pyodide]
+config-settings = "build-dir=build setup-args=--cross-file=$PWD/tools/ci/emscripten/emscripten.meson.cross setup-args=-Dblas=none setup-args=-Dlapack=none"
+
```

Comment:
This will use lapack-lite. Is there a better BLAS implementation available?

Reply:
It's supposed to use `lapack-lite` here (for now at least), that's what is happening in Pyodide in-tree as well. It's possible that will switch to `openblas` in the future, but it doesn't really matter here (performance isn't a high prio for things like interactive docs).

Reply:
Happy to make it try to use OpenBLAS or any alternative BLAS vendor in the future (not for now, though – indeed not a priority)